pkg/notes/options/options.go

```diff
@@ -151,5 +178,63 @@ func (o *Options) ValidateAndFinish() error {
 		}
 	}
 
+	// Create the record dir
+	if o.RecordDir != "" {
+		logrus.Info("using record mode")
+		return o.writeOptions()
+	}
+
 	return nil
 }
+
+func (o *Options) recoverForReplay() error {
+	file, err := ioutil.ReadFile(filepath.Join(o.ReplayDir, optionsFile))
+	if err != nil {
+		return errors.Wrapf(err, "unable to read options file")
+	}
+	previousReplayDir := o.ReplayDir
+	previousOutput := o.Output
+
+	if err := json.Unmarshal(file, o); err != nil {
+		return errors.Wrapf(err, "unable to recover options file")
+	}
+	o.Output = previousOutput
+	o.RecordDir = "" // record and replay together is not suppored
+	o.ReplayDir = previousReplayDir
+	o.gitCloneFn = git.CloneOrOpenGitHubRepo
+	return nil
+}
```

Comment:
I think I understand why this (forcing the same options as were used in the recorded session) happens, but I think the least we should do then is to make that really clear to the user, in other words log a warning that basically all your options/flags will be ignored.

Reply:
I will remove the options de-/serialization to avoid confusion here.